src/main/scala/org/constellation/primitives/EdgeDAO.scala

```diff
@@ -81,11 +81,13 @@ class ThreadSafeMessageMemPool() extends StrictLogging {
 
 import constellation._
 // TODO: wkoszycki this one is temporary till (#412 Flatten checkpointBlock in CheckpointCache) is finished
+case class PendingDownloadException(id: Id)
+    extends Exception(s"Node [${id.short}] is not ready to accept blocks from others.")
 case object MissingCheckpointBlockException extends Exception("CheckpointBlock object is empty.")
 case class MissingHeightException(cb: CheckpointBlock)
     extends Exception(s"CheckpointBlock ${cb.baseHash} height is missing for soeHash ${cb.soeHash}.")
-case class PendingAcceptance(cb: CheckpointBlock)
-    extends Exception(s"CheckpointBlock: ${cb.baseHash} is already pending acceptance phase.")
+case class PendingAcceptance(baseHash: String)
```

Comment:
I'd consider changing `baseHash` to sth like `cbHash`. baseHash is widely used within the codebase but you wanted the baseHash of checkpoing block only